packages/shared/lib/services/sync/config/deploy.service.ts

```diff
@@ -180,10 +205,6 @@ export async function deploy({
             deployResults.push(...result);
         }
 
-        for (const id of idsToMarkAsInactive) {
-            await switchActiveSyncConfig(id);
```

Comment:
can we keep this function instead of duplicating the raw sql, passing the `trx` to the function?